operation/client_handler.go

```diff
@@ -165,6 +166,8 @@ func (client *clientHandler) handleBasicCommand(command architecture.Command) ar
 		}
 	case architecture.TOUCH:
 
+	case architecture.QUIT:
+		client.conn.Close()
```

Comment:
I haven't written a test so I can't confirm that yet.  My thinking was:

1. Closing the `client.conn` connection breaks the loop at [line 80](https://github.com/vimukthi-git/beanstalkg/pull/62/files/b83b7f4fdc65f5ef1c7503d0ca9d770d97674879#diff-24e84e23d62ef57f58d78401a6110fe7L78).
2. Breaking that loop sends a `true` to the `exit` channel, which is received on [line 107](https://github.com/vimukthi-git/beanstalkg/pull/62/files/b83b7f4fdc65f5ef1c7503d0ca9d770d97674879#diff-24e84e23d62ef57f58d78401a6110fe7L107).
3. Receiving from the `exit` channel executes `return`, which exits the `startSession` function.

Do I have that wrong?